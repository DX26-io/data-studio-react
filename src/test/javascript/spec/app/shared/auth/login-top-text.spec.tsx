import React from 'react';
import { shallow } from 'enzyme';
import Typography from '@material-ui/core/Typography';
import LoginTopText from 'app/modules/login/login-top-text';

describe('Login Top Text', () => {
  let mountedWrapper;

  const wrapper = () => {
    if (!mountedWrapper) {
      mountedWrapper = shallow(<LoginTopText />);
    }
    return mountedWrapper;
  };

  beforeEach(() => {
    mountedWrapper = undefined;
  });

  it('Renders Login Top Text', () => {
    const component = wrapper();
    expect(component).toMatchSnapshot();
    const typography = component.find(Typography);
    expect(typography.length).toEqual(1);
  });

});
