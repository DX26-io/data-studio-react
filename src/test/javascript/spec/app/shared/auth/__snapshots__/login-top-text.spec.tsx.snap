// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login Top Text Renders Login Top Text 1`] = `
<WithStyles(ForwardRef(Typography))
  className="dx26-font-bold"
  style={
    Object {
      "position": "absolute",
      "right": "3%",
      "top": "4%",
    }
  }
  variant="h6"
>
  DX26
</WithStyles(ForwardRef(Typography))>
`;
